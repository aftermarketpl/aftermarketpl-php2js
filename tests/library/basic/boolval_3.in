$a = 0;
return boolval($a);
