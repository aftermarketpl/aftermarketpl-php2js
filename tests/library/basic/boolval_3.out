a = 0;
return Boolean(a);
